chsdi/lib/helpers.py

```diff
@@ -289,15 +292,29 @@ def parse_box2d(stringBox2D):
     return map(float, extent.split(' '))
 
 
-def center_from_box2d(box2D):
+def is_box2d(box2D):
     # Bottom left to top right only
-    if box2D[0] > box2D[2] or box2D[1] > box2D[3]:
+    if box2D[0] > box2D[2] or box2D[1] > box2D[3] or len(box2D) != 4:
         raise ValueError('Invalid box2D.')
+    return True
+
+
+def extend_box2d(box2D, distance):
+    if is_box2d(box2D):
+        return [
+            box2D[0] - distance,
+            box2D[1] - distance,
+            box2D[2] + distance,
+            box2D[3] + distance
+        ]
 
-    return [
-        box2D[0] + ((box2D[2] - box2D[0]) / 2),
-        box2D[1] + ((box2D[3] - box2D[1]) / 2)
-    ]
+
+def center_from_box2d(box2D):
+    if is_box2d(box2D):
```

Comment:
In case if this being False, client code gets None and will probably fail, is it?